SConstruct

```diff
@@ -84,11 +84,12 @@ def update_rpm_version(version, tag):
             spec[line_num] = "Release:       {}%{{?relval}}%{{?dist}}\n".\
                              format(release)
         if line == "%changelog\n":
+            cmd = 'rpmdev-packager'
             try:
-                packager = subprocess.Popen(
-                    'rpmdev-packager', stdout=subprocess.PIPE).communicate(
-                    )[0].strip().decode('UTF-8')
+                pkg_st = subprocess.Popen(cmd, stdout=subprocess.PIPE) # nosec
+                packager = pkg_st.communicate()[0].strip().decode('UTF-8')
```

Comment:
Please note:
Have to break the subprocess command into small commands. Since # comment inside the subprocess.Popen() will be treat as a parameter and causing issue. 
Need to double define "packager = .." statement for both Except and non-Except conditions.